src/main/java/edu/hawaii/its/groupings/configuration/OotbSecurityConfig.java

```diff
@@ -0,0 +1,102 @@
+package edu.hawaii.its.groupings.configuration;
+
+import edu.hawaii.its.groupings.access.DelegatingAuthenticationFailureHandler;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.authentication.configuration.AuthenticationConfiguration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+import org.springframework.security.web.csrf.CookieCsrfTokenRepository;
+import org.springframework.ws.config.annotation.DelegatingWsConfiguration;
+
+import java.util.List;
+
+import static org.springframework.security.config.Customizer.withDefaults;
+import static org.springframework.security.web.util.matcher.AntPathRequestMatcher.antMatcher;
+
+@EnableWebSecurity
+@Configuration
+@ConditionalOnProperty(name = "grouping.api.server.type", havingValue = "OOTB")
+public class OotbSecurityConfig {
+
+    private static final Log logger = LogFactory.getLog(OotbSecurityConfig.class);
+    @Value("${grouping.api.server.type}")
+    private String serverType;
+
+    @Value("${ootb.default.user.name}")
+    private String name;
+
+    @Value("${ootb.default.user.username}")
+    private String username;
+
+    @Value("${ootb.default.user.authorities}")
+    private List<String> authorities;
```

Comment:
I think you should put all these variables into the OotbCustomUserDetails class. Then the variables are all in one place rather than being split between two classes. Then at the `ootbStaticUserAuthenticationFilter()` bean, It can simply be: `return new OotbStaticUserAuthenticationFilter();`